browser.jsx

```diff
@@ -285,14 +290,14 @@ function UserTokenDisplay (props) {
       {
         props.refreshToken
           ? <div>
-            Refresh token: <input type="text" readonly value={props.refreshToken} />
+            Refresh token: <input type="text" readOnly value={props.refreshToken} />
           </div>
           : <div>
             Refresh token: (None, you selected a temporary duration)
           </div>
       }
       <div>
-        Access token: <input type="text" readonly value={props.accessToken} />
+        Access token: <input type="text" readOnly value={props.accessToken} />
```

Comment:
Would it make sense to have the `refreshToken` and `accessToken` copyable here too?

Reply:
Yes, sorry i forgot about those !